be/src/io/cache/block_file_cache.cpp

```diff
@@ -2211,6 +2213,14 @@ bool BlockFileCache::try_reserve_during_async_load(size_t size,
     return !_disk_resource_limit_mode || removed_size >= size;
 }
 
+void BlockFileCache::clear_need_update_lru_blocks() {
+    std::vector<FileBlockSPtr> buffer;
+    buffer.reserve(1024);
+    while (_need_update_lru_blocks.try_dequeue_bulk(buffer.data(), buffer.capacity())) {
+        buffer.clear();
```

Comment:
this `clear()` does not make sense, it does not clear anything since buffer.size() is always 0 because L2218 is `reserve()`
do nothing if you just intend to use a tmp buffer for bulk dequeue.